pages/cloudflare/howtos/image.mdx

```diff
@@ -40,6 +40,13 @@ export default function cloudflareLoader({ src, width, quality }: ImageLoaderPro
 }
 ```
 
+<Callout type="warning">
+  This simple loader does not respect [Next.js
+  remotePatterns](https://nextjs.org/docs/pages/api-reference/components/image#remotepatterns). You should
```

Comment:
these links appear broken in the rich diff - is that just because of the callout and not being parsed properly or are they broken locally as well? unfortunate that there's no pr environments to check

![image](https://github.com/user-attachments/assets/89356c3a-8547-4d2e-9639-31f3c679749a)

Reply:
Working fine in dev

<img width="1174" alt="image" src="https://github.com/user-attachments/assets/4edc7f57-a1cf-4a81-b2a1-cbb5d5cb404e" />